charts/alfresco-repository/values.yaml

```diff
@@ -46,14 +46,37 @@ configuration:
     # -- Name of a pre-existing secret
     # TODO: secret documentation
     existingSecret: null
+  messageBroker:
+    # -- Message Broker URL
+    url: null
+    # -- Username to authenticate to the message broker
+    username: null
+    # -- Password to authenticate to the message broker
+    password: null
+    # -- Existing secret and their keys where to find the message broker
+    # username & password.
```

Comment:
```suggestion
```

pretty much the same of what's documented below - and don't generate the ugly object entry in helm docs

Reply:
it slipped through